sdk/src/client/api/block_builder/input_selection/manual.rs

```diff
@@ -39,7 +39,7 @@ impl<'a> ClientBlockBuilder<'a> {
 
         if let Some(inputs) = &self.inputs {
             for input in inputs {
-                let output_with_meta = self.client.get_output(input.output_id()).await?;
+                let output_with_meta = self.client.get_output_with_metadata(input.output_id()).await?;
```

Comment:
is this maybe better named `input_with_meta`?

Reply:
Tbh I won't bother, Thoralf is removing this file in his PR.

Reply:
No, it returns an output (which will just be used as input)

Reply:
okay